src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -82,7 +82,7 @@ class NetworkOPsImp final
         bool const admin;
         bool const local;
         FailHard const failType;
-        bool applied;
+        bool applied{false};
```

Comment:
This uniform initialization stuff makes my eyes bleed. It's just so jarring. Unless there's some really cool, esoteric advantage to this, can we just use `bool applied = false` please?

Reply:
Fixed in `bd4cc1711 [fold] Misc fixes:`